requirements.txt

```diff
@@ -1,5 +1,5 @@
 aiofiles>=22.0,<24.0
-altair>=4.2.0,<6.0
+altair>=5.3.0,<6.0
```

Comment:
Curious, why upgrade?

Reply:
actually not necessary, will revert. altair 4.0 has no type hinting which was annoying internally but this makes no difference for the end user and also, altair 5.0 is still allowed